LOCALIZATION_CONTRIBUTORS

```diff
@@ -178,7 +178,7 @@ Murat Boy
 Doğukan Tuna
 Utku Birkan
 Davut Emre TAŞAR
-
```

Comment:
Carlos Alberto Reyes Salcedo

Reply:
Hi @CUANTICOCARLOS, please open a different pull request to add your name to the list. Thank you for your interest.